fs/accounting/stats.go

```diff
@@ -37,12 +39,130 @@ type StatsInfo struct {
 
 // NewStats cretates an initialised StatsInfo
 func NewStats() *StatsInfo {
-	return &StatsInfo{
+	s := &StatsInfo{
 		checking:     newStringSet(fs.Config.Checkers),
 		transferring: newStringSet(fs.Config.Transfers),
 		start:        time.Now(),
 		inProgress:   newInProgress(),
 	}
+
+	rc.Add(rc.Call{
+		Path:  "stats/transfers",
+		Fn:    s.bwStats,
+		Title: "Get transfer stats",
+		Help: `
+Show statistics for all transfers.
+`,
+	})
+
+	rc.Add(rc.Call{
+		Path:  "stats/reset",
+		Fn:    s.bwResetStats,
+		Title: "Reset transfer stats",
+		Help: `
+Resets the global transfer statistic counters.
+`,
+	})
+
+	return s
+}
+
+// reset statistics as requested by rc stats/reset
+func (s *StatsInfo) bwResetStats(in rc.Params) (out rc.Params, err error) {
+	out = make(rc.Params)
+
+	s.mu.Lock()
+	s.bytes = 0
+	s.errors = 0
+	s.checks = 0
+	s.transfers = 0
+	s.deletes = 0
+	s.start = time.Now()
+	s.mu.Unlock()
+
+	out["status"] = "ok"
+	out["message"] = "Global transfer statistic counters were reset to Zero"
+	return out, nil
+}
+
+// create map with statistics for rc stats/transfers
+func (s *StatsInfo) bwStats(in rc.Params) (out rc.Params, err error) {
```

Comment:
This should probably be `rcStats` to fit in with the naming conventions elsewhere